src/LowerTriangularArrays/LowerTriangularArrays.jl

```diff
@@ -11,10 +11,13 @@ import KernelAbstractions
 # NUMERICS
 import LinearAlgebra: tril!
 
+export AbstractSpectrum, Spectrum, resolution, truncation
+ 
 export LowerTriangularMatrix, LowerTriangularArray
 export eachharmonic, eachmatrix
```

Comment:
In RingGrids, given that the `grid` is now inherently 2D we have
- `eachgridpoint` returning a 2D iterator
- `eachindex` for the 3D iterator
- `eachlayer` returning an ND iterator for vertical, time, ...
- `eachring` for the latitude iterator 

Can we revisit that `eachharmonic`, `eachindex`, `eachmatrix`, `eachorder` (<- does not exist yet? but possible with the new indices in `Spectrum`) are equivalent for consistency?

Reply:
Ok, I look into it. But just mentioning: with the introduction of the KA kernels, we won't use these iterators a lot. Most of these loops will be replaced by a kernel.

Reply:
Okay, added that now